src/options.rs

```diff
@@ -49,6 +51,16 @@ fn parse_range_option(input: &str) -> IResult<&str, UsingOption> {
         |t: (&str, &str)| UsingOption::Range(t.1.to_string()))(input)
 }
 
+fn parse_colnames_option(input: &str) -> IResult<&str, UsingOption> {
```

Comment:
Would you mind adding a test for this option? Very basic and simple. This would also document what is considered as allowed input for this option.

Also adding example to README.md would be nice, but I could do this myself.

Reply:
I added some tests. I tried to add to the readme too, but I wasn't sure how to do explain it clearly.